apps/src/code-studio/pd/workshop_dashboard/workshop_dashboard.jsx

```diff
@@ -124,18 +125,33 @@ export default class WorkshopDashboard extends React.Component {
             <Route
               path="workshops/new"
               breadcrumbs="Workshops,New Workshop"
-              component={NewWorkshop}
+              component={props => (
```

Comment:
instead of passing through the token as a separate prop, would it be possible to add it to the [reducers](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/pd/workshop_dashboard/reducers.js) and update the constructor to set the mapbox token via a reducer?